content/blog/2023/03/camel-observation/index.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Camel Micrometer Observation: Observability with Micrometer"
+date: 2023-03-28
+draft: false
+authors: [marcingrzejszczak]
+categories: ["Camel Observation", "Features"]
+preview: "Micrometer Observation: how to configure Distributed Tracing and Metrics"
+---
+
+The Spring Observability Team has added native observability support for Spring Applications with Spring Framework 6 and Spring Boot 3. You can read more about the feature in [the Spring blog](https://spring.io/blog/2022/10/12/observability-with-spring-boot-3). This blog post will explain what Observability is, how it's done with Micrometer Observation and how you can add observability to your Camel projects.
+
+## What is Observability?
+
+In our understanding, it is _"how well you can understand the internals of your system by examining its outputs"_. We believe that the interconnection between metrics, logging, and distributed tracing gives you the ability to reason about the state of your system in order to debug exceptions and latency in your applications. You can watch more about what we think observability is in https://tanzu.vmware.com/developer/tv/enlightning/10/[this episode of Enlightning with Jonatan Ivanov].
+
+With Spring Boot `3.0.0` release we've added numerous autoconfigurations for improved metrics with [Micrometer](https://micrometer.io/) and new distributed tracing support with [Micrometer Tracing](https://micrometer.io/docs/tracing)  (formerly [Spring Cloud Sleuth](https://spring.io/projects/spring-cloud-sleuth)).
```

Comment:
As you're talking with the "we" pronoun, it seems this is something Camel team has done. You better change the phrases to show Spring team did that work.